livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py

```diff
@@ -141,6 +144,10 @@ def __init__(
         api_key = api_key or os.environ.get("ELEVEN_API_KEY")
         if not api_key:
             raise ValueError("ELEVEN_API_KEY must be set")
+        
+        if language is not None and model != "eleven_turbo_v2_5":
+            language = None
+            logger.warning("Excluding 'language' as it is only supported for the 'eleven_turbo_v2_5' model")
```

Comment:
Let's do nothing by default? I think 11labs will throw an error if we use a different language no?
Otherwise lgtm

Reply:
Yea it will throw an error, will remove this